recipes/pysam/0.7.7/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: pysam
+  version: "0.7.7"
+source:
+  fn: pysam-0.7.7.tar.gz
+  url: https://pypi.python.org/packages/source/p/pysam/pysam-0.7.7.tar.gz
+  sha256: c9f3018482eec99ee199dda3fdef2aa7424dde6574672a4c0d209a10985755cc
+build:
+  number: 0
+requirements:
+  build:
+    - python <3.0.0
+    - setuptools
+    - cython
+    - gcc  # [linux]
+    - llvm # [osx]
+    - zlib
+  run:
+    - python <3.0.0
+    - libgcc # [linux]
+    - zlib
+test:
+  # Python imports
+  imports:
+    - pysam
+#    - pysam.include
+#    - pysam.include.htslib
```

Comment:
I basically copied it from 0.8.3. The pysam/include directory in the package does not contain any files. These modules are therefore not included in the package and the comments I copied from 0.8.3 were probably only meant to document this.

What should I do? Leave them? Remove them?